packages/mermaid/src/docs/syntax/gitgraph.md

```diff
@@ -513,18 +513,19 @@ Here, we have changed the default main branch name to `MetroLine1`.
 
 ## Orientation (v10.3.0+)
 
-In Mermaid, the default orientation is Left to Right. The branches are lined vertically.
+In Mermaid, the default orientation is for commits to run from **left to right** and for branches to be stacked on top of one another.
+
+You can set this explicitly with `LR:` after `gitGraph`.
 
```

Comment:
Here i suggest to add what was there at the beginning

```md
In `LR` orientation commits run from **left to right** and branches are stacked on top of one another.
```

Reply:
Looks good! Please, do not resolve conversations by yourself, if they are unresolved it makes it easier for a reviewer to get back to the comments and revise eveything